mix.exs

```diff
@@ -40,7 +40,7 @@ defmodule NervesHub.MixProject do
       {:phoenix_channel_client, "~> 0.3"},
       {:websocket_client, "~> 1.3"},
       {:jason, "~> 1.0"},
-      {:nerves_runtime, ">= 0.6.5"},
+      {:nerves_runtime, ">= 0.6.5", app: Mix.env() == :prod},
```

Comment:
I haven't used the `:app` key here before. Is `nerves_runtime` only included if `MIX_ENV=prod`? I'd think that we'd want it included if the mix target isn't "host", right?

Reply:
This is a bit of a hack. It includes it, but doesn't start it. Ideally we'd want it to be a target only dep, but then we'd need to try guessing the target.

it could be `app: Mix.Project.config()[:target] != "host` also i think

Reply:
Could you do this a little bit different? The problem with `nerves_runtime` not working well when run on the host is a common problem. Rather than put the solution here and have people copy/paste it to every other project, could you put the solution in `nerves_runtime`?

If you look at `nerves_runtime`, there's a config param called `:enable_syslog` that's part way to what you need. As a start, I might change `:enable_syslog` to be more encompassing and turn off uevent processing and load in fake KV pairs. For example, the config could look something like:

```elixir
config :nerves_runtime,
  target: "host",
  kv: %{"nerves_serial_number" => "1234",
    "nerves_fw_active" => "a",
    "a.nerves_fw_uuid" => "123455892920",
    "nerves_hub_cert" => "something",
    "nerves_hub_key" => "secret"}
```
What do you think?